Assets/GothicVR/Scripts/Player/Running/MovementTypeController.cs

```diff
@@ -0,0 +1,203 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.XR;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Manager;
+
+namespace GVR.Player
+{
+    public class MovementTypeController : MonoBehaviour
+    {
+        //No UI icon for now 
+        //public Canvas runningCanvas;
+        public GameObject leftHand;
+        public GameObject rightHand;
+        public CharacterController character;
+
+        public static float walking_speed;
+        public static float running_speed;
```

Comment:
Since we need to handle movement stuff also in ClimbProvider, it would be better to have "running speed" set as a constant in ConstantManager